BuildResidentialHPXML/measure.rb

```diff
@@ -1594,6 +1594,12 @@ def arguments(model)
     arg.setDefaultValue(false)
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('water_heater_is_shared_system', true)
+    arg.setDisplayName('Water Heater: Is Shared System')
+    arg.setDescription('Whether the water heater is a shared system.')
```

Comment:
Since you're not asking for the number of units served by the shared water heater and are assuming it serves all units in the building, which is reasonably fine, I suggest adding that to the description. Something like:
'Whether the water heater is a shared system. If true, assumed to served all the units in the building.'